### DISCUSSION TITLE
Can I transfer my plot files to another chia farming wallet/app?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#2210] [Q&A]
I was wondering, is it possible to transfer or plot files on different pcs and the transfer them to another one? If yes, does it work by simply copying it?

**Comment by dorkmo** [comment]
yup, copy and paste the plots to the new drive. be sure to add the location in the plots directory. gotta use the same 24 key too.

**Reply by GeorgePapageorgakis** [reply]
Can I have multiple wallets with the same keys on different pcs farming as one whole wallet?
